### PR TITLE
Style and add useState & event handler to Request an inspection popup

### PR DESCRIPTION
**Pull request by AshleeTan**

### CODE REVIEW

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+              <Address>{props.address}</Address>
+              <SubTitle>Inspection day and time</SubTitle>
+              <DayAndTimeContainer>
+                <Date type="date" placeholder="Day" /><Time type="time" placeholder="Time" />
+              </DayAndTimeContainer>
+              <CheckboxContainer>
+                <Checkbox type="checkbox" />
+                <CheckboxContent>
+                  <SubTitle>I can't find a suitable day or time</SubTitle>
+                  <RemarksTitle>One of our agents will get in touch to arrange another time</RemarksTitle>
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:226]: We can change it to `label` as well like 
```js
const TimeNotSuitableRemarksLabel = styled.label`
  display: block;
  font-family: "PangeaRegular";
  font-size: 11px;
  font-weight: 300;
  color: rgb(97, 97, 97);
`;
```

Comment (JakeLin) [reply src/components/property-detail/hero/RequestInspectionPopupScreen.js:226]: Then add `for="time-no-suitable-label"` to make the checkbox selectable.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+              <Title>Book an Inspection</Title>
+              <Address>{props.address}</Address>
+              <SubTitle>Inspection day and time</SubTitle>
+              <DayAndTimeContainer>
+                <Date type="date" placeholder="Day" /><Time type="time" placeholder="Time" />
+              </DayAndTimeContainer>
+              <CheckboxContainer>
+                <Checkbox type="checkbox" />
+                <CheckboxContent>
+                  <SubTitle>I can't find a suitable day or time</SubTitle>
```
[the remark this answers is not in the document]
Comment (JakeLin) [reply src/components/property-detail/hero/RequestInspectionPopupScreen.js:225]: Then add `for="time-no-suitable-label"` to make the checkbox selectable.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+            <BrandingBar listingCompany={props.listingCompany} height="40" />
+            <ContentContainer>
+              <Title>Book an Inspection</Title>
+              <Address>{props.address}</Address>
+              <SubTitle>Inspection day and time</SubTitle>
+              <DayAndTimeContainer>
+                <Date type="date" placeholder="Day" /><Time type="time" placeholder="Time" />
+              </DayAndTimeContainer>
+              <CheckboxContainer>
+                <Checkbox type="checkbox" />
```
[the remark this answers is not in the document]
Comment (JakeLin) [reply src/components/property-detail/hero/RequestInspectionPopupScreen.js:223]: Then add `id="time-no-suitable-label"` to make the checkbox can be selected from labels.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+  margin: 10px 0 5px 0;
+`;
+
+const SubmitButtonContainer = styled.button`
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:184]: Can name it `SubmitButton`

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+  margin-left: 0;
+`;
+
+const CheckboxContent = styled.div`
+  margin-left: 8px;
+`;
+
+const Divider = styled.div`
+  margin: 0 auto;
+  width: 370px;
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:125]: can use `width: 100%;` instead

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+
+const DayAndTimeContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-top: 8px;
+`;
+
+const Date = styled.input`
+  width: 38%;
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:80]: Use `  box-sizing: border-box;
  width: calc(50% - 6px);` is better.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+`;
+
+const Time = styled.input`
+  width: 38%;
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:94]: Use `  box-sizing: border-box;
  width: calc(50% - 6px);` is better.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+`;
+
+const PersonalDetailsInputContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+
+const PersonalDetailsInput = styled.input`
+  width: 38%;
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:138]: Can use `  box-sizing: border-box;
  width: calc(50% - 6px);` is better.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+  margin: 10px 0 5px 0;
+`;
+
+const DropdownListContainer = styled.input`
+  box-sizing: border-box;
+  width:100%;
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:154]: need a space before `100%`

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 10]
```diff
@@ -0,0 +1,252 @@
...
+              </CheckboxContainer>
+              <Divider />
+              <SubTitle>Your personal details</SubTitle>
+              <PersonalDetailsInputContainer>
+                <PersonalDetailsInput type="text" placeholder="First Name" />
+                <PersonalDetailsInput type="text" placeholder="Last Name" />
+                <PersonalDetailsInput type="email" placeholder="Email" />
+                <PersonalDetailsInput type="number" placeholder="Mobile" />
+              </PersonalDetailsInputContainer>
+              <DropdownListContainer type="text" placeholder="Current Situation" />
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:237]: Rename to `DropdownList`

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 11]
```diff
@@ -0,0 +1,252 @@
...
+  border-width: 2px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
+  letter-spacing: 0.6px;
+  text-align: left;
+`;
+
+const CommentContainer = styled.input`
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:168]: Rename to `CommentTextarea` and use `styled.textarea` instead.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 12]
```diff
@@ -0,0 +1,252 @@
...
+  box-sizing: border-box;
+  width: 100%;
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:170]: Add `height: 100px;` to make it taller.

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js [hunk 13]
```diff
@@ -0,0 +1,252 @@
...
+const PersonalDetailsInput = styled.input`
+  width: 38%;
+  border-radius: 4px;
+  border-color: rgb(195, 200, 206);
+  border-style: solid;
+  border-width: 1px;
+  padding: 13px 16px;
+  font-size: 13px;
+  font-weight: 300;
+  color: rgb(155, 155, 155);
```
Comment (JakeLin) [inline src/components/property-detail/hero/RequestInspectionPopupScreen.js:146]: change the color to `color: #333f48;`
and add the below code to change the placeholder text color
```
  &::placeholder {
    color: #333f4866;
  }
```

**Review by JakeLin — COMMENTED** [review]
Very good PR, please have a look at the comments as well, thanks

File: src/components/property-detail/hero/RequestInspectionPopupScreen.js (hunk 1 above)
Comment (AshleeTan) [reply src/components/property-detail/hero/RequestInspectionPopupScreen.js:226]: Wrap SubTitle & RemarksTitle by <label for="time-not-suitable-label"></label>

**Review by JakeLin — APPROVED** [review]
Well done

**Pull request merged**